specs/binding-filesystem.spec/src/main/resources/META-INF/zilla/filesystem.idl

```diff
@@ -21,7 +21,10 @@ scope filesystem
         READ_CHANGES,
         CREATE_PAYLOAD,
         WRITE_PAYLOAD,
-        DELETE_PAYLOAD
+        DELETE_PAYLOAD,
+        CREATE_DIRECTORY,
+        DELETE_DIRECTORY,
+        LIST_CONTENTS
```

Comment:
Perhaps `_PAYLOAD` should become `_FILE` in the above capabilities now that we have `_DIRECTORY` capabilities?
Then `LIST_CONTENTS` might just be `READ_DIRECTORY`?

`READ_CHANGES` seems like it might become `READ_FILE_CHANGES`
`READ_EXTENSION` seems like it might become `READ_METADATA` so then we could support `HEAD` requests for files or directories.